### DISCUSSION TITLE
inference模型转nb、onnx模型报错

### DISCUSSION THREAD

**Discussion by renshujiajia** [PaddlePaddle/PaddleOCR#12385] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

系统环境/System Environment：Linux ocrdevelop 5.11.0-27-generic 导出识别模型时都是 KeyError: 'predict'  #29~20.04.1-Ubuntu SMP Wed Aug 11 15:58:17 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
版本号/Version：Paddle： PaddlePaddle-gpu 2.6.0 PaddleOCR2.7.0： 问题相关组件/Related components：ppocrv3训练及导出
运行指令/Command Code：paddle_lite_opt --model_file=./ch_PP-OCRv3_rec_slim_infer/inference.pdmodel --param_file=./ch_PP-OCRv3_rec_slim_infer/inference.pdiparams --optimize_out=./ch_PP-OCRv3_rec_slim_opt --valid_targets=arm --optimize_out_type=naive_buffer
完整报错/Complete Error Message：Loading topology data from inference.pdmodel
Loading params data from inference.pdiparams
Model is successfully loaded!
[F 1/16 10:59:55.518 ...e-Lite/lite/core/model/general/op_desc.h:118 GetAttr] Check failed: it != attrs().end(): No attributes called beta found for swish
Aborted (core dumped)
我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by renshujiajia** [comment]
官方提供的en_PP-OCRv3_rec_train.tar和en_PP-OCRv3_rec_slim_train.tar直接转inference模型成功，但是转nb或者onnx存在相同问题

**Comment by kevinzezel** [comment]
Same error on my custom finetune model rec_v4.

**Comment by GreatV** [comment]
更新一下paddle2onnx版本，现在这个问题应该修复了

**Reply by GreatV** [reply]
这个是由于paddle版本升级，paddle2onnx之间没有进行兼容。

**Reply by kevinzezel** [reply]
Thanks for the quick response! I'm using Paddle v2.6.0. Should I downgrade to which version?

**Reply by GreatV** [reply]
Hi @kevinzezel No need to downgrade PaddlePaddle, just use the latest version Paddle2ONNX.

**Reply by kevinzezel** [reply]
Hi @GreatV! I'm using this command to convert my finetune model to an opt model nb to run in paddle lite:

./opt_linux_aarch64 --model_file=/root/src/pretrain_models/rec_ppocr_v4_NEW_v1_test/inference.pdmodel --param_file=/root/src/pretrain_models/rec_ppocr_v4_NEW_v1_test/inference.pdiparams --optimize_out=/root/src/pretrain_models/rec_ppocr_v4_NEW_v1/inference_opt --valid_targets=arm  --optimize_out_type=naive_buffer

Where does paddle2onnx come in?

**Reply by GreatV** [reply]
@kevinzezel  My mistake, you are right, you may need to downgrade the paddle version in this scenario.

**Reply by kevinzezel** [reply]
Thank you!
